contributing/documentation/overview.rst

```diff
@@ -136,7 +136,7 @@ changes and push them:
 
     # ... do your changes
 
-    $ git push
+    $ git push origin improve_install_chapter
```

Comment:
You are right, since it's related to the *first contribution* and following the previous steps, `git push` is enough.
My thought while adding this was: if `push.default` is set to `matching` all the local branches are pushed to their matching ones on the remote. Anyway I will revert this change, since it's an edge case.